--- scottfree-1.14.orig/Makefile
+++ scottfree-1.14/Makefile
@@ -4,11 +4,16 @@
 CC	=	gcc
 #
 #
-all	:	ScottCurses
+all	:	scottfree
 
 ScottCurses.o:	ScottCurses.c Scott.h
 
-ScottCurses:	ScottCurses.o
-	$(CC) ScottCurses.o -o ScottCurses -lcurses -ltermcap
+scottfree:	ScottCurses.o
+	$(CC) $(LDFLAGS) ScottCurses.o -o scottfree -lcurses
 
+install:
+	mkdir -p $(DESTDIR)/usr/bin
+	install -m 755 scottfree $(DESTDIR)/usr/bin
 
+clean:
+	rm -f ScottCurses.o scottfree
